### PR TITLE
Updated to kubernetes 1.13.2

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/4709.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -33,31 +33,31 @@ required = [
   name = "github.com/docker/distribution"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 2]
```diff
@@ -33,31 +33,31 @@ required = [
...
   name = "github.com/giantswarm/apiextensions"
 
 [[constraint]]
   branch = "master"
   name = "github.com/giantswarm/backoff"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 3]
```diff
@@ -33,31 +33,31 @@ required = [
...
   name = "github.com/giantswarm/certs"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 4]
```diff
@@ -33,31 +33,31 @@ required = [
...
   name = "github.com/giantswarm/crdstorage"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 5]
```diff
@@ -33,31 +33,31 @@ required = [
...
   name = "github.com/giantswarm/e2e-harness"
 
 [[constraint]]
   branch = "master"
   name = "github.com/giantswarm/e2etemplates"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 6]
```diff
@@ -81,19 +81,19 @@ required = [
   name = "github.com/giantswarm/micrologger"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 7]
```diff
@@ -81,19 +81,19 @@ required = [
...
   name = "github.com/giantswarm/operatorkit"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

**Review by TheoBrigitte — APPROVED** [review]

File: Gopkg.toml [hunk 8]
```diff
@@ -81,19 +81,19 @@ required = [
...
   name = "github.com/giantswarm/randomkeys"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

File: Gopkg.toml [hunk 9]
```diff
@@ -81,19 +81,19 @@ required = [
...
   name = "github.com/giantswarm/statusresource"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.13.2"
```
Comment (xh3b4sd) [inline Gopkg.toml]: TODO

**Review by rossf7 — APPROVED** [review]

**Pull request merged**